docs/source/guides/multisig_native_assets_api.md

```diff
@@ -420,5 +421,5 @@ message SendTxResponse {
 - **credentials** (these are the credentials generated by the PSG Services Self Serve UI after the user has created a "log in" and paid for use of the services.) [new_user_guide](./new_user_guide.md)
 
 #### SendTxResponse
-- In the case of `success` the call will return the `SendTx` object which contains infromation about Transaction and Native Asset
+- In the case of `success` the call will return the `TxSubmitted` which is indicator that transaction has been submitted.
```

Comment:
This is incorrect now. It will return `TxSubmitted` as a first element of the stream, which will be followed by `SendTx` objects